docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-feature-integration.md

```diff
@@ -28,6 +28,7 @@ Install the required modules:
 ```bash
 composer require spryker-feature/marketplace-product: "dev-master" --update-with-dependencies
```

Comment:
please use release tag

Reply:
command that install feature should be with `dev-master` according to "Feature integration guides"